### PR TITLE
ARCH-323 Full Link Postcard

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
-  Added new display mode for postcard paragraphs to allow linking the entire card.

# Needed By (Date)
- August 8th

# Urgency
- High

# Steps to Test
1. checkout this branch
1. `comopser update --prefer-source`
1. `blt drupal:sync --site=archaeology --sync-files`
1. create a basic page with a row containing 2+ postcards
1. configure those postcards to have a read more link and choose the "vertical linked card" display 
1. verify the entire card is linked to the button you supplied
1. verify both cards are the same height (even if there is a gap at the bottom of one)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by boznik** [comment]
This is good to go by me. @josephgknox can you review?

File: docroot/themes/humsci/su_humsci_theme/patterns/molecules/vertical-link-card/vertical-link-card.html.twig [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{% set attributes = attributes.addClass('vertical-link-card') %}
+
+<div{{ attributes }}>
+  {% if button %}
+  <a href="{{ button|render|striptags|trim }}" title="{{ title|render|striptags|trim }}">
+    {% endif %}
```
[the remark this answers is not in the document]
Comment (pookmish) [reply docroot/themes/humsci/su_humsci_theme/patterns/molecules/vertical-link-card/vertical-link-card.html.twig]: thats what phpstorm did 🤷‍♂️

**Comment by pookmish** [comment]
Oh hey there code climate!

**Comment by sherakama** [comment]
Yeah, finally got the permissions sorted out for this org. Let me know if you want to turn it off for this repo if it is too noisy. If you want it on, you may want to add a `.codeclimate.yml` file.

**Comment by josephgknox** [comment]
This looks good to me functionally, too. I will push up front-end stuff here now.

**Comment by josephgknox** [comment]
@pookmish @boznik this is good from my perspective. LMK if there are any changes to what I added.

**Pull request merged**